src/igtcloud/client/tools/cli.py

```diff
@@ -26,16 +27,33 @@ def cli():
 @click.option('--domain', default=None, help='Overwrites the environment setting')
 @click.option('--user', default=None, help='Username')
 @click.option('--ext', default=None, help='filter on specific file extension (i.e.: \'.txt\')')
-@click.option('--start', default=None, help='start date (YYYY-MM-DD) of date range filter. Default value is 1900-01-01.')
-@click.option('--end', default=None, help='end date (YYYY-MM-DD) of date range filter. Default value is 9999-12-31.')
-@click.option('--category', default=['files'], multiple=True, type=click.Choice(['files', 'dicom', 'annotations'],
-                                                                                case_sensitive=False),
-              help='Categories of files to download')
+@click.option(
+    '--start',
+    default=None,
+    help='start date (YYYY-MM-DD) of date range filter. Default value is 1900-01-01. Ignored if --project-files is set.'
+)
+@click.option(
+    '--end',
+    default=None,
+    help='end date (YYYY-MM-DD) of date range filter. Default value is 9999-12-31. Ignored if --project-files is set.'
+)
+@click.option(
+    '--category',
+    default=['files'],
+    multiple=True,
+    type=click.Choice(['files', 'dicom', 'annotations'], case_sensitive=False),
+    help='Categories of files to download. Ignored if --project-files is set.'
+)
 @click.option('--include-modified-date', flag_value=True,
               help='Set the accurate file last modified date metadata instead of disk creation date')
+@click.option(
+    '--project-files',
+    flag_value=True,
+    help='Only download the project files. If not set, only the study files are downloaded.'
```

Comment:
This means you cannot download project files and institute files at the same time? Is that desired?

Reply:
Yes, this is the desired behavior. The user has an option to download only project files or studies and not everything in one go.